### ISSUE TITLE
Improve exception messages when using QueryFile?

### ISSUE THREAD

**Issue by cowboy** [vitaly-t/pg-promise#166]
Is there any way that the filename can be included in an exception message when an exception is thrown using a query from a file passed to `QueryFile`?

It would help when debugging!

**Comment by vitaly-t** [comment]
It is included!

**Comment by cowboy** [comment]
Hmmn.. this is the error I just saw:

```
error: column reference "name" is ambiguous
    at [object Object].Connection.parseE (/Users/cowboy/repos/bocoup/expertisebot/node_modules/pg/lib/connection.js:539:11)
    at [object Object].Connection.parseMessage (/Users/cowboy/repos/bocoup/expertisebot/node_modules/pg/lib/connection.js:366:17)
    at Socket.<anonymous> (/Users/cowboy/repos/bocoup/expertisebot/node_modules/pg/lib/connection.js:105:22)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at TCP.onread (net.js:523:20)
```

**Comment by vitaly-t** [comment]
And how is this related to [QueryFile](http://vitaly-t.github.io/pg-promise/QueryFile.html)?

**Comment by cowboy** [comment]
That was the exception I got when I made a query using the `QueryFile` object. I'm not seeing any errors when creating the `QueryFile` object, but I am when querying the db using that object.

**Comment by vitaly-t** [comment]
I can't see what is going on from a piece of the error log that's not even referencing the library...

**Comment by cowboy** [comment]
Sorry, I apologize for not being clear, let me try again. When I do this, I see that stack trace because of an error in `my_query.sql`.

```js
const pgp = pgPromise(options);
const db = pgp(config);
const queries = utils.enumSql(__dirname, {}, file => {
  return QueryFile(file, {
    minify: true,
    debug: true,
  });
});
db.query(queries.myQuery).then(r => console.log(r));
```

Is there any way for the error message to include `my_query.sql` somewhere, to aid in debugging?

**Comment by vitaly-t** [comment]
The error you are seeing is not related to QueryFile at all.

**Comment by cowboy** [comment]
Ok.
